apps/src/music/player/MusicLibrary.ts

```diff
@@ -125,8 +125,8 @@ export default class MusicLibrary {
     // Combine the JSON-specified folders into one flat list of folders.
     this.folders = [
```

Comment:
could the type of `this.folders` be `(SoundFolder | SoundFolderInstrument | SoundFolderKit)[]` so this cast isn't needed?

Can also create a separate type (`LibraryFolder`?) that `= SoundFolder | SoundFolderInstrument | SoundFolderKit` if that helps